server/tests/e2e/activities.e2e-spec.ts

```diff
@@ -0,0 +1,200 @@
+// noinspection DuplicatedCode
+
+import * as request from 'supertest';
+import { get, skipTest, takeTest } from '../util/test-util';
+import { MapStatus, MapCreditType, MapType } from '../../src/common/enums/map.enum';
+import { ActivityTypes } from '../../src/common/enums/activity.enum';
+import { PrismaService } from '../../src/modules/repo/prisma.service';
+import { AuthService } from '../../src/modules/auth/auth.service';
+import { ActivityDto } from '../../src/common/dto/user/activity.dto';
+
+describe('activities', () => {
+    let user1, user2, map1, activity1, activity2, activity3;
+
+    beforeAll(async () => {
+        const prisma: PrismaService = global.prisma;
+
+        //test user1
+        user1 = await prisma.user.create({
+            data: {
+                steamID: '65465432154',
+                alias: 'User 1',
+                roles: { create: { verified: true, mapper: true } }
+            }
+        });
+
+        //test user2
+        user2 = await prisma.user.create({
+            data: {
+                steamID: '754673452345',
+                alias: 'User 3',
+                roles: { create: { mapper: true } }
+            }
+        });
+
+
+        map1 = await prisma.map.create( {
+            data: {
+                name: 'maps_test' + Math.floor(Math.random() * 100000000),
```

Comment:
Lets change naming here, not just copy verbatim from maps test. Also looks like this could just be a hardcoded name?

Reply:
Actually looks like map1 is unneeded